website/src/app/blog/firezone-1-0/readme.mdx

```diff
@@ -0,0 +1,243 @@
+  src="/images/nostalgia-show-hn.png"
+  alt="Show HN"
+  width={1412}
+  height={168}
+  className="mx-auto shadow rounded"
+/>
+
+When we [launched on Hacker News](https://news.ycombinator.com/item?id=28683231)
+nearly two years ago, we never envisioned Firezone to be more than a simple tool
+deploying your own WireGuard-based VPN server.
+
+Fast-forward
+[4,500 GitHub stars](https://github.com/firezone/firezone/stargazers), a
+[Y Combinator backed funding round](https://techcrunch.com/2022/03/30/ycombinator-open-source-startups-winter-22-demo-day/),
+and [130 releases](https://github.com/firezone/firezone/releases) later --
+Firezone has now grown into something more than just a self-hosted tool to
+manage your WireGuard configurations.
+
+We now count over <strong>3,000</strong> Firezone instances running in the wild
+(possibly much more -- we allow users to
+[disable telemetry](/docs/reference/telemetry)) securing private networks for
+hobbyists, schools, non-profits, and businesses with hundreds of employees.
+
+<Image
+  src="/images/wai-7-2-23.png"
+  alt="weekly active instances"
+  width={600}
+  height={600}
+  className="mx-auto shadow rounded border border-neutral-200"
+/>
+
+To be clear, Firezone is successful in large part because WireGuard itself is
+successful. In an industry brimming with enterprise security bloatware and
+endless acronyms, WireGuard's a breath of fresh air. Every issue I've thought I
+had with it turned out to be user error. How we ever got by without it is a
+mystery to me.
+
+But I could go on and on about WireGuard's strengths. Let's back up for a minute
+-- what is a VPN, and why is one needed at all? To answer that we'll need to go
+back to the formation of the Internet itself.
+
+#### The purpose of a VPN
+
+You see, the early Internet had only a handful of entities connected to it --
+connecting to the Internet was expensive, after all. Typically only banks,
+universities, and other large institutions could justify the cost.
+
+So when you connected your organization to the Internet and began receiving
+packets, it was clear which entity it was from based on its allocated IPv4
+address range. Since there were so few entities connected, it was clear who you
+communicated with. It was clear who to contact (and blame) in case any issues
+arose.
+
+But, as access to the Internet became cheaper, more types of entities could
+afford to connect. As more entities connected, the number of resources on the
+Internet grew, and its
+[value increased quadratically](https://en.wikipedia.org/wiki/Metcalfe%27s_law).
+Soon, all types of entities wanted to connect -- local governments, schools,
+small businesses. Internet Service Providers began offering connections to mere
+individuals. Eventually there were so many entities on the Internet that
+identifying who you were communicating with was no longer trivial. Since you
+couldn't easily know who you were talking to, you couldn't always trust them not
+to misbehave.
```

Comment:
There's a lot of negation in the sentence:
```
Since you couldn't easily know who you were talking to, you couldn't always trust them not
to misbehave.
```

What would think about changing this to something along the lines of:
```
Since you couldn't easily know who you were talking to, you couldn't always trust them to behave.
```

Reply:
Thanks, fixed!